Add the payload parameter description (#21050)
hashicorp/vault: website/content/docs/enterprise/license/utilization-reporting.mdx

HashiCorp collects the following utilization data as JSON payloads:

- `payload_version` - The version of this payload schema
- `license_id` - The license ID for this product
- `product` - The product that this contribution is for
- `product_version` - The product version this contribution is for
- `export_timestamp`- The date and time for this contribution
- `snapshots` - An array of snapshot details. A snapshot is a structure that

>   represents a single - data collection.

Reviewer: ```suggestion
  represents a single data collection
```

   - `snapshot_version` - The version of the snapshot package that produced this
     snapshot
   - `snapshot_id` - A unique identifier for this particular snapshot
   - `process_id` - An identifier for the system that produced this snapshot
   - `timestamp` - The date and time for this snapshot
   - `schema_version` - The version of the schema associated with this snapshot
   - `service` - The service that produced this snapshot (likely to be product
     name)
   - `metrics` - A map of representations of snapshot metrics contained within
     this snapshot
      - `key` - The key name associated with this metric
      - `kind` - The kind of metric (feature, counter, sum, or mean)
      - `mode` - The mode of operation associated with this metric (write or
        collect)
      - `labels` - The labels associated with each collected metric

>          - `entity` - The counter of tokens generated for a unique client identifier
>          - `nonentity` - The counter of tokens without an entity attached

Reviewer: ```suggestion
         - `entity` - The sum of tokens generated for a unique client identifier
         - `nonentity` - The sum of tokens without an entity attached
```

> - `metadata` - Optional product-specific metadata
>    - `billing_start` - The unique ID of the Replicated license
>    - `cluster_id` - The Salesforce Opportunity ID associated with the license
> 

Reviewer: ```suggestion
- `metadata` - Optional product-specific metadata
   - `billing_start` - The billing start date associated with the reporting cluster (license start date if not configured)
   - `cluster_id` - The cluster UUID as shown by `vault status` on the reporting cluster
```
